content/en/docs/reference/glossary/aggregation-layer.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Aggregation Layer
+id: aggregation-layer
+date: 2018-10-08
+full_link: /docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/
+short_description: >
+  A mechanism to extend Kubernetes by aggregating the existing API Server with additional Kubernetes-style APIs servers
```

Comment:
Not sure if "APIs servers" is good English. Maybe we should say "API servers" instead?